pytket/extensions/aqt/multi_zone_architecture/architecture.py

```diff
@@ -0,0 +1,83 @@
+from enum import Enum
+from typing import Union, Dict, List
+
+from pydantic import BaseModel
+
+
+class EdgeType(str, Enum):
+    Right = "Right"
+    Left = "Left"
+
+
+class ConnectionType(str, Enum):
```

Comment:
Can you add some documenting comments to this classes and functions in this file?